app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -264,6 +264,12 @@ def raw_connect_v4(options = {})
       # Get the timeout from the configuration:
       timeout, = ems_timeouts(:ems_redhat, options[:service])
 
+      # The constructor of the SDK expects a list of certificates, but that list can't be empty, or contain only 'nil'
+      # values, so we need to check the value passed and make a list only if it won't be empty. If it will be empty then
+      # we should just pass 'nil'.
+      ca_certs = options[:ca_certs]
+      ca_certs = [ca_certs] if ca_certs
```

Comment:
`options[:ca_certs]` is the content of the `endpoints.certificate_authority` column. That is a single string that may contain multiple concatenated certificates, that is why the name is plural, but the content is either `nil` or one string.